lib/suma/member/front_attributes.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require "suma/front"
+
+class Suma::Member::FrontAttributes
+  def initialize(member)
+    @member = member
+  end
+
+  def upsert_contact
+    return self.create_contact if self.contact_id.blank?
+
+    self.update_contact
+    self.add_contact_handles
+  end
+
+  def create_contact
+    contact = Suma::Front.client.create_contact!(self.contact_body.merge(handles: self.contact_handles))
+    self.update_contact_id(contact.fetch("id"))
+    contact
+  end
+
+  def update_contact
```

Comment:
Part of updating a contact is updating its handles, so do that in this method rather than requiring another call.